hist/hist/src/TH1.cxx

```diff
@@ -150,17 +159,46 @@ Histograms may also be created by:
   -  making a projection from a 2-D or 3-D histogram, see below
   -  reading an histogram from a file
 
-  When an histogram is created, a reference to it is automatically added
+ When a histogram is created, a reference to it is automatically added
  to the list of in-memory objects for the current file or directory.
  This default behaviour can be changed by:
 ~~~ {.cpp}
-       h->SetDirectory(0);          for the current histogram h
-       TH1::AddDirectory(kFALSE);   sets a global switch disabling the reference
+       h->SetDirectory(0);          // for the current histogram h
+       TH1::AddDirectory(kFALSE);   // sets a global switch disabling the referencing
+~~~
+ When the histogram is deleted, the reference to it is removed from
+ the list of objects in memory.
+ When a file is closed, all histograms in memory associated with this file
+ are automatically deleted.
+
+### Labelling axes
+
+ Axis titles can be specified in the title argument of the constructor.
+ They must be separated by ";":
+~~~ {.cpp}
+        TH1F* h=new TH1F("h", "Histogram title;X Axis;Y Axis;Z Axis", 100, 0, 1);
```

Comment:
```suggestion
        TH1F* h=new TH1F("h", "Histogram title;X Axis;Y Axis", 100, 0, 1);
```

Reply:
Even though there is a Z axis even in TH1 I wouldn't show it as an example.

Reply:
Very valid point. Thanks! 👍